docker/benchmarks/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# The ideal base image would be "microsoft/dotnet-nightly:2.1.0-preview1-runtime-deps-stretch".
+# However, the wrk package is currently broken in "debian:stretch" (4.0.2-1+b1), but fixed in
+# "debian:buster" (4.0.2-2).  For now, it's easier to use "debian:buster" as the base image and 
+# manually install the .NET Core dependencies, rather than using "microsoft/dotnet-nightly" and
+# compiling wrk from source.
+
+FROM debian:buster
+
+# .NET Core dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        libc6 \
+        libcurl3 \
+        libgcc1 \
+        libgssapi-krb5-2 \
+        libicu57 \
+        liblttng-ust0 \
+        libssl1.0.2 \
+        libstdc++6 \
+        libunwind8 \
+        libuuid1 \
+        zlib1g \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update \
```

Comment:
The idea was the .NET Core dependencies are temporary, so I just copied the whole `apt-get` statement from the .NET Core Dockerfile.  I'll move these as a section in the other `apt-get` statement, it will be just as easy to remove later.